### PR TITLE
Add draft status to assessment form

### PR DESCRIPTION
**Pull request by iamtheluckyest**
Adds a "Draft" status to Assessment Forms so that users can edit assessment metadata and assign individual sections, etc. before sharing these with assignees.

### CODE REVIEW

File: src/assessmentForm.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /** AssessmentForm status types */
 export enum AssessmentFormStatus {
+  /** The form has been created but has not been shared with assignees */
+  Draft = 'DRAFT',
```
Comment (abrantesarthur) [inline src/assessmentForm.ts:4]: Don't we need a migration for this?

Comment (iamtheluckyest) [reply src/assessmentForm.ts:4]: We will, but only after it merges to privacy-types I think. Then I'll update the package version in the monorepo and write a migration.

Comment (abrantesarthur) [reply src/assessmentForm.ts:4]: gotcha! Looked too quickly and didn't notice this was privacy-types!

**Review by abrantesarthur — APPROVED** [review]
thank you!

**Pull request merged**